src/Mod/Sketcher/App/SketchObject.cpp

```diff
@@ -2516,12 +2514,13 @@ void SketchObject::transferFilletConstraints(int geoId1, PointPos posId1, int ge
         std::vector<int> deleteme;
         for (int i = 0; i < int(constraints.size()); i++) {
             const Constraint* c = constraints[i];
-            if (c->Type == Sketcher::Distance || c->Type == Sketcher::Equal) {
-                bool line1 = c->First == geoId1 && c->FirstPos == PointPos::none;
-                bool line2 = c->First == geoId2 && c->FirstPos == PointPos::none;
-                if (line1 || line2) {
-                    deleteme.push_back(i);
-                }
+            if (c->Type != Sketcher::Distance && c->Type != Sketcher::Equal) {
+                continue;
+            }
+            bool line1 = c->First == geoId1 && c->FirstPos == PointPos::none;
+            bool line2 = c->First == geoId2 && c->FirstPos == PointPos::none;

```

Comment:
You should use the new getGeoId/setGeoId etc everywhere you refactor the code. First and FirstPos etc are deprecated.